plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -70,7 +73,37 @@ def _coerce_to_cartesian_si(pos):
     return pos_out
 
 
-class Tracker:
+class _SyntheticRadiographySaveRoutine(AbstractSaveRoutine):
+    ARRAY_KEYS = ["x", "y", "v", "x0", "y0", "v0"]
```

Comment:
Should save everything in the Tracker's `results_dict`

Reply:
Do you mean that everything inside of `results_dict` should be saved to the HDF file-- say as an attribute by default?

Reply:
Yep - for the charged particle radiography case we want everything in that dict